ee/ucp/admin/install/cloudproviders/install-on-azure.md

```diff
@@ -45,6 +45,7 @@ to successfully deploy Docker UCP on Azure:
   Configuration](#considerations-for-ipam-configuration).
 - All UCP worker and manager nodes need to be attached to the same Azure
   Subnet.
+- All UCP Nodes NIC's (Network Interface) must have IP forwarding enabled.
```

Comment:
Do we have a link to how to enable this in Azure?

Reply:
Looks like this is not a requirement at all.